homeassistant/components/thread/websocket_api.py

```diff
@@ -180,6 +180,9 @@ async def ws_list_datasets(
                 "preferred_border_agent_id": dataset.preferred_border_agent_id,
                 "preferred_extended_address": dataset.preferred_extended_address,
                 "source": dataset.source,
+                "tlv": dataset.tlv,
```

Comment:
The idea was/is that you don't need the raw TLV hex string when showing a list of datasets to the user, so the TLV is hidden behind a separate call, to be fetched when the user wants to import a dataset.